README.md

```diff
@@ -3,29 +3,57 @@
 [![Build Status](https://travis-ci.org/brettminnie/breakfast-serializer.svg)](https://travis-ci.org/brettminnie/breakfast-serializer)
 [![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/brettminnie/breakfast-serializer/badges/quality-score.png?b=develop)](https://scrutinizer-ci.com/g/brettminnie/breakfast-serializer/?branch=develop)
 [![Dependency Status](https://www.versioneye.com/user/projects/55378b007f43bcd88900033d/badge.svg?style=flat)](https://www.versioneye.com/user/projects/55378b007f43bcd88900033d)
+[![Coverage Status](https://coveralls.io/repos/brettminnie/breakfast-serializer/badge.svg?branch=master&service=github)](https://coveralls.io/github/brettminnie/breakfast-serializer?branch=master)
 
-A (hopefully) backwards compatible replacement for the other well known serializer
+A replacement for the other well known serializer, initially we were aiming for backwards compatibility, however it seemed
+more appropriate to develop a lightweight alternative. We are forgoing the depth of features now to offer something that is
+easy to configure and works well without any configuration.
 
-Currently will only support json and do a full depth/breadth recursion, we'll build on this slowly
+### Installation
 
-Example
+`$ composer require brettminnie/breakfast-serializer` 
```

Comment:
Might be worth putting a version on the end of this? `$ composer require brettminnie/breakfast-serializer ...VERSION...`